examples/advanced/gnn/code/pyg_sage.py

```diff
@@ -0,0 +1,23 @@
+import torch.nn as nn
```

Comment:
Please add license header to this file.

If this file is adapted from other sources, we can cite it as well.

Reply:
thanks, added